bindings/node.js/bindings.ts

```diff
@@ -0,0 +1,186 @@
+export interface Blst {
+  SecretKey: SecretKeyConstructor;
+  P1_Affine: P1_AffineConstructor;
+  P2_Affine: P2_AffineConstructor;
+  P1: P1Constructor;
+  P2: P2Constructor;
+  PT: PTConstructor;
+  Pairing: PairingConstructor;
+}
+
+// blst.hpp types
+
+type bool = boolean;
+type size_t = number;
+type app__string_view = string | Uint8Array;
+type DST = string;
```

Comment:
Some further reading shows that suggestion to use capitalized types is not appropriate in the context. It's suggested that `boolean`, `number`, `string` are true TS basic types. As well as `bigint` that will be needed now. Apparently `Uint8Array` falls out of the pattern and there is no corresponding "basic" type. Oh well, sorry about distraction. Question about `DST` still stands though...

Yet, as for Uint8Array. Since `type byte[s] = Uint8Array` is suggested, why not use `string | byte[s]`?

Reply:
I initially used types that mimic the C types for ease of maintenance by C devs, but it's best to just follow the Typescript standard. We don't need the DST type indeed. Uint8Array is not strictly a basic type but could be considered as such. It's the cross-platform standard to represent bytes